Web.md

```diff
@@ -14,6 +14,12 @@ than if the module was JavaScript.
 More concretely, the following is a list of points of contact between WebAssembly
 and the rest of the Web platform that have been considered:
 
+## The `Wasm` object
+
+WebAssembly is exposed to the Web through a `Wasm` object, whose API is designed
```

Comment:
Hah, so weird, I distinctly remember writing a longer comment.  Oh I know what happened, I wrote &lt;script&gt; without escaping the &lt;.  Sorry about that.

I meant to end with: ... WebAssembly exposed through &lt;script type=module&gt;, `import`, and Worker constructors that took a module URL.